[DOCS] Host Isolation notifications (#1470)
elastic/security-docs: docs/management/admin/host-isolation-ov.asciidoc

To isolate a host from an endpoint:

. Go to *Manage -> Endpoints*, then either:

>     * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Isolate host* in the endpoint details flyout.

Reviewer: ```suggestion
    * Select the appropriate endpoint in the *Endpoint* column, and click *Take action -> Isolate host* in the endpoint details flyout.
```
Petty but figured this might be a bit more readable since 'then' is used in the previous sentence. If you happen to agree there's another instance further down the same page.

>     * Click the *Actions* icon (*...*) for the appropriate endpoint, then select *Isolate host*.

Reviewer 2: Discussed in our design leads meeting that these are actually menus. 
```suggestion
    * Click the *Actions* menu (*...*) on the appropriate endpoint, then select *Isolate host*.
```

. Enter a comment describing why you’re isolating the host (optional).
. Click *Confirm*.

> After the host is successfully isolated, an *Isolated* status is added to the endpoint. Any active end user receives a notification that the computer has been isolated from the network:

Reviewer 3: ```suggestion
After the host is successfully isolated, an *Isolated* status is added to the endpoint. Active end users receive a notification that the computer has been isolated from the network:
```

> After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any user currently on the host receives a notification that the computer has been isolated from the network:

Reviewer: ```suggestion
After the host is successfully isolated, the *Agent Status* changes to *Isolated*. Any user currently on the host receives a notification that the computer has been isolated from the network:
```
Seems redundant to say that a status is added to agent status, also if you're looking on the agent table then Isolated appears under Agent status, not next to it.

Author: ```suggestion
After the host is successfully isolated, an *Isolated* status is added to the endpoint. Any user currently on the host receives a notification that the computer has been isolated from the network:
```

> After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any user currently on the host receives a notification that the computer has been isolated from the network:

Reviewer 2: Just a suggestion here. Debating b/t "any active end user" and "an active end user." 
```suggestion
After the host is successfully isolated, an *Isolated* status is added next to *Agent Status*. Any active end user on the host receives a notification that the computer has been isolated from the network:
```

Author: + "active end user". I think "An active end user" feels a bit nonspecific, so that's why I originally went with "Any...."

. Go to *Investigate -> Cases*, then click on the appropriate case to view the case activity its details.
. Find the appropriate alert, then click the *Show alert details* button (*>*). The alert details flyout opens.
. From the alert details flyout, click *Take action -> Release host*.
. Enter a comment describing why you're releasing the host (optional).
. Click *Confirm*.

To release a host from an endpoint:

. Go to *Manage -> Endpoints*, then either:

>     * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Release host* in the endpoint details flyout.

Author: ```suggestion
    * Select the appropriate endpoint in the *Endpoint* column, and click *Take action -> Release host* in the endpoint details flyout.
```

>     * Click the *Actions* icon (*...*) for the appropriate endpoint, then select *Release host*.

Reviewer 2: Same comment as above about menu vs. icon.

Author: 👍 
```suggestion
    * Click the *Actions* menu (*...*) on the appropriate endpoint, then select *Release host*.
```

. Enter a comment describing why you're releasing the host (optional).
. Click *Confirm*.

> After the host is successfully released, the *Isolated* status is removed from the endpoint. Any active end user receives a notification that the computer has been reconnected to the network:

Reviewer 3: ```suggestion
After the host is successfully released, the *Isolated* status is removed from the endpoint. Active end users receive a notification that the computer has been reconnected to the network:
```

> After the host is successfully released, the *Isolated* status is removed from the *Agent Status*. Any user currently on the host receives a notification that the computer has been reconnected to the network:

Reviewer: ```suggestion
After the host is successfully released, the *Agent Status* changes back to *Healthy*. Any user currently on the host receives a notification that the computer has been reconnected to the network:
```
I guess it could change back to something else but I don't think that would confuse anyone.

Author: @benironside This isn't quite correct; the "Isolated" label doesn't replace anything or vice versa; it's added or removed. So an endpoint could be both "Healthy" and "Isolated" at the same time (per the screenshot at the top of the page). But I agree that the phrasing is still inexact, so:
```suggestion
After the host is successfully released, the *Isolated* status is removed from the endpoint. Any user currently on the host receives a notification that the computer has been reconnected to the network:
```

> After the host is successfully released, the *Isolated* status is removed from the *Agent Status*. Any user currently on the host receives a notification that the computer has been reconnected to the network:

Reviewer 2: ```suggestion
After the host is successfully released, the *Isolated* status is removed from the *Agent Status*. Any active end user on the host receives a notification that the computer has been reconnected to the network:
```

To view the host’s isolation details:

> . Go to *Manage -> Endpoints*, then select the appropriate host name in the *Endpoint* column. The endpoint details flyout opens.

Reviewer: ```suggestion
. Go to *Manage -> Endpoints*, then click the host's name in the *Endpoint* column. The endpoint details flyout opens.
```
